src/components/styles/BookForm.module.css

```diff
@@ -1,3 +1,58 @@
 * {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/155657987-52b0c12a-9281-4b89-bc8e-70f838908633.png)

Therefore:

- [ ] Please, make the form widths the same width as the book items as in the given design.
- [ ] Kindly add the semi-transparent vertical lines that appear on the given design.
- [ ] Please, match the font styling on the design as much as you can.  

Look:

Your Implementation | Zeplin Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/155657561-517cec41-d303-4370-bca0-519a909051d1.png) | ![image](https://user-images.githubusercontent.com/57421823/155657599-c0375813-8f87-4324-a3cd-d3ea96333be7.png)

You can achieve the required styling by going to the Zeplin design, selecting an element, and looking to the right:  almost all the exact CSS code is there:


![image](https://user-images.githubusercontent.com/57421823/155657772-fe94051e-8f82-40c0-bf6d-c5d646b282a1.png)

That way, your project will look like the Zeplin design, and you will be able to provide a software CSS solution to a given business requirement. 💪